### ISSUE TITLE
No imaging library available. ImageIO module was not correctly installed

### ISSUE THREAD

**Issue by malekjuby** [domlysz/BlenderGIS#617]
# **Blender and OS versions**

# **Describe the bug**

# **How to Reproduce**

# **Error message**

**Comment by zgrzzx** [comment]
我也是

**Comment by arildj78** [comment]
Blender 3.4.1
Windows 11
Brand new laptop. Installed Blender and BlenderGIS for the first time on this machine.
Norton Security firewall (trial edition) factory installed.

The addon seemed to install correctly but pressing
**GIS** -> **Web geodata** -> **Basemap**
gave the error:   `No imaging library available. ImageIO module was not correctly installed`

Also, trying to search for new Spatial Reference Systems to add, returned no results.
No feedback is given to the user to check firewall settings. I ended on a wild goose chase to look for ImageIO.

After turning off the firewall, and restarting Blender, everything worked.

**Comment by sifrani** [comment]
I have the same problem, did you solve it?

**Comment by arildj78** [comment]
Yes. The firewall was the issue. After disabling it everything worked as expected.
